src/control/common/storage/pb_types.go

```diff
@@ -101,7 +97,7 @@ func (ncs NvmeControllers) ctrlrDetail(buf *bytes.Buffer, c *ctlpb.NvmeControlle
 		c.Pciaddr, c.Serial, c.Model, c.Fwrev, c.Socketid)
 
 	for _, ns := range c.Namespaces {
-		fmt.Fprintf(buf, "\t\tNamespace: id:%d capacity:%d\n", ns.Id, ns.Capacity)
+		fmt.Fprintf(buf, "\t\tNamespace: id:%d capacity:%dGB\n", ns.Id, ns.Capacity)
```

Comment:
This will break (or at least look odd) for SSDs with larger capacities, right? Seems like you could use the bytesize representation by multiplying the capacity times 1000*1000*1000. I'd probably declare that value as a const, e.g. `const gigabytes float64 = 1000 * 1000 * 1000`, and then you can just use it like this: `bytesize.New(float64(ns.Capacity)*gigabytes)`.